index.html

```diff
@@ -843,6 +843,64 @@ <h2>Skills</h2>
    
     </section>
     <!--about me section ends here -->
+     
+              
+    <!--contact me   -->
+    <section class="contact">
+
+      <div class="contact-container">
+
+          <div class="contact-container-desktop">
+           
+            <h2 id="contact">Contact me</h2>
+            
+            <p class="contact-P contact-P-desktop">
+              If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+              I’d love to help with it!
+            </p>
+
+              <!--  contact form section  -->
+              <form class="contact-form" action="https://formspree.io/f/xaygnnvv" method="POST">
+                
+                <ul class="form-section">
+
+                  <li>
+                    
+                  <input type="text" name="name" class="input-desktop" maxlength="30" placeholder="Melkamu Tessema"
+                      required>
+  
+                  </li>
+  
+                  <li>
+
+                  <input type="email" name="email" class="input-desktop" placeholder="melkamu372@gmail.com " required>
+  
```

Comment:
hello @joel-kalema thank you for your time I did not understand  [OPTIONAL] It could be better to give the placeholder the value name inside of Melkamu Tessema for your input name and mail inside of melkamu372@gmail.com for the mail input.
do you mean change plaeholder to like enter full name Or ?? befor I were doing that but the figima design use name  the place holder 
![image](https://github.com/melkamu372/Portfolio-mobile-first/assets/47281626/d11a3bae-c8d3-4225-ac7e-624e4f807b5e)
but If no problem I change it.

Reply:
Yes @melkamu372 , it will be something like `placeholder= "name"` and `placeholder= "mail"`